pasteit/routes/functions.js

```diff
@@ -0,0 +1,47 @@
+var redis = require("redis");
+var client = redis.createClient();
+var uuid = require("uuid4");
+
+
+module.exports = {
+    newPaste : function(req) {
+        var newPaste = module.exports.createPaste(req);
+        var hash = uuid();
+        module.exports.addNewPaste(hash, newPaste);
+
+        return hash;
+    },
+
+    createPaste : function(req) {
+        var newPaste = {
+            "language" : req.body.language,
+            "paste" : req.body.paste,
+            "mime" : req.body.mime
+        }
+
+        return newPaste;
+    },
+
+    addNewPaste : function(hash, newPaste) {
+        client.hmset(hash, newPaste);
```

Comment:
client is a bad name for this variable. It should be "redisClient" so it's clear we're inserting into redis here.